style.css

```diff
@@ -258,7 +262,7 @@ html {
 }
 
 .project1-btn:hover {
-  background-color: #ee5100;
+  background-color: rgb(231, 220, 175);
   transform: translateY(3px);
 }
```

Comment:
- [ Optional] On hover the buttons should be white and the text color orange, but in your project, everything is white so the text is disappearing? Kindly change the color text of the buttons on hover.